ios/Rainbow Stickers/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
Where does this pull from? Want to make sure I update correctly on release

Reply:
you just need to edit this from XCode. It has been moved to some variable, but it doesn't matter